hice_backend/settings.py

```diff
@@ -2,9 +2,20 @@
 import os
 import sys
 from pathlib import Path
+import boto3
 
 load_dotenv()
 
+AWS_CLOUDWATCH_LOGGER_ACCESS_KEY = os.getenv("AWS_CLOUDWATCH_LOGGER_ACCESS_KEY")
+AWS_CLOUDWATCH_LOGGER_SECRET_KEY = os.getenv("AWS_CLOUDWATCH_LOGGER_SECRET_KEY")
+AWS_CLOUDWATCH_LOGGER_REGION_NAME = os.getenv("AWS_CLOUDWATCH_LOGGER_REGION_NAME")
+
+boto3_logs_client = boto3.client(
+    "logs",
+    region_name=AWS_CLOUDWATCH_LOGGER_REGION_NAME,
+    aws_access_key_id=AWS_CLOUDWATCH_LOGGER_ACCESS_KEY,
+    aws_secret_access_key=AWS_CLOUDWATCH_LOGGER_SECRET_KEY,
+)
```

Comment:
Can you make it so that this only runs in production and non-testing environments?

We shouldn't need to add the keys to the repo.